test/cypress/integration/21-collective.create.test.js

```diff
@@ -27,4 +27,16 @@ describe('create a collective', () => {
       'The slug bees-are-neat is already taken. Please use another slug for your collective.',
     );
   });
+
+  it('Can create a collective with tags', () => {
+    cy.getByDataCy('ccf-category-picker-button-community').click();
+    cy.getByDataCy('ccf-form-name').type('Bees are vicious');
+    cy.getByDataCy('ccf-form-description').type('I do not like them');
+    cy.get('[data-cy=styled-input-tags-open]').click();
+    cy.get('[data-cy=styled-input-tags-add-suggestion-meetup]').click();
+    cy.get('[data-cy=styled-input-tags-input]').type('opencollective-tag{enter}{esc}');
+    cy.get('[data-cy=ccf-form-tags]').contains('opencollective-tag');
+    cy.get('[data-cy="custom-checkbox"').click();
+    cy.get('[data-cy=ccf-form-submit]').click();
```

Comment:
Great to have a test for this! We should make sure to update it to make sure tags are properly added once we have https://github.com/opencollective/opencollective/issues/5422

Reply:
Just to note that this is taken care of at, https://github.com/opencollective/opencollective-frontend/pull/7772